driver_test.go

```diff
@@ -2948,6 +2948,9 @@ func TestRowsColumnTypes(t *testing.T) {
 
```

Comment:
Put this code here:

```go
if t.Failed() {
    return
}
```

Reply:
like this code？
I tested it and the code does seem more appropriate.
```
		for i := range values {
			if types[i] == nil {
				t.Failed()
				return
			}
			values[i] = reflect.New(types[i]).Interface()
		}
```

Reply:
No. Just put my code here.

I think `types[i] == nil` happens only when test is already failed.